LoopKit/HealthKitSampleStore.swift

```diff
@@ -25,6 +25,8 @@ public enum HealthKitSampleStoreResult<T> {
 public class HealthKitSampleStore {
     /// Describes the source of an update notification. Value is of type `UpdateSource.RawValue`
     public static let notificationUpdateSourceKey = "com.loopkit.UpdateSource"
+    
+    private let observerQueryUpdateHandlerQueue = DispatchQueue(label: "com.loopkit.HealthKitSampleStore.observerQueryUpdateHandlerQueue", qos: .utility)
```

Comment:
[nit] Not sure it matters much, but each of the 3 stores will have a dispatch queue with the same label.

Reply:
Good point; I can fix that.